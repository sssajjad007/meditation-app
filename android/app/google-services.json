{
  "project_info": {
    "project_number": "189336514254",
    "project_id": "denj-7a257",
    "storage_bucket": "denj-7a257.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:189336514254:android:b11eb2826989b16883ea9f",
        "android_client_info": {
          "package_name": "com.denjapp.android"
        }
      },
      "oauth_client": [
        {
          "client_id": "189336514254-qkmfh2ifiq5j2cado5hi7t3gadjclrp7.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyBueVwFYBJwdd16j96p0iqLHe0hgD6JAQQ"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "189336514254-qkmfh2ifiq5j2cado5hi7t3gadjclrp7.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}

