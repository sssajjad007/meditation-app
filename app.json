{
  "name": "Denj",
  "displayName": "Denj",
  "expo": {
    "name": "Denj",
    "slug": "Denj",
    "version": "1.0.0",
    "scheme": "denj",
    "assetBundlePatterns": [
      "**/*"
    ],
    "android": {
      "package": "com.denjapp.android"
    },
    "ios": {
      "bundleIdentifier": "com.karafs.denj"
    },
    "plugins": [
      "sentry-expo"
    ]
  }
}
