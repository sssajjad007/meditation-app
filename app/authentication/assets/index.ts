export const LogoImage = require("./Logo.png");
export const AuthImage = require("./AuthImage.png");
