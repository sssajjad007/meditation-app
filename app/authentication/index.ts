export { Authentication } from "./screens/Auth";
