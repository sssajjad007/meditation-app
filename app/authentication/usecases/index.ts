export { register } from "./register";
export { login } from "./login";
export { completeSignup } from "./completeSignup"