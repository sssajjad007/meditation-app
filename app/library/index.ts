export { Input } from "./Input";
export { Button } from "./Button";
export { THEME } from "./theme";
export { Touchable } from "./Touchable";
export {
  Caption,
  Headline,
  Paragraph,
  Subheading,
  Title,
  Typography,
} from "./Typography";
export { CustomBackdrop } from "./CustomBackdrop";
export { Tap } from "./Tap";
export { Container } from "./Container";
// export { openCamera, openCropper, openGallery } from "./cropPicker";
export { request, refresh, uploadFile } from "./network";
export { Timer } from "./Timer";
export { toString } from "./toString";
export { secureStorage, storage } from "./storage";
export { Checkbox } from "./CheckBox";
export { Radio } from "./Radio";
export { SelectButton } from "./SelectButton";
export { version, marketFlag } from "./Version";
export * from "./Icon";
