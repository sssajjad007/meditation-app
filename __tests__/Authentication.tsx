import React from "react";
import renderer from "react-test-renderer";
import { Authentication } from "../app/authentication";

test("renders correctly", () => {
  const tree = renderer.create(<Authentication />).toJSON();
  expect(tree).toMatchSnapshot();
});
