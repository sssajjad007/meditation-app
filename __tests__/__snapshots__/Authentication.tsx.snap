// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaView
  style={
    Object {
      "backgroundColor": "#F9FAFC",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 26,
        "height": 293.5,
        "justifyContent": "center",
        "width": 750,
      }
    }
  >
    <Image
      resizeMode="stretch"
      source={1}
      style={
        Object {
          "height": 80,
          "width": 80,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 133.5,
        "justifyContent": "center",
        "width": 750,
      }
    }
  >
    <Text
      mode="mediumHeadline22"
      style={
        Array [
          Array [
            Object {
              "fontFamily": "Dana-Medium",
              "fontSize": 22,
            },
            Object {
              "bottom": 36,
            },
          ],
        ]
      }
    >
      شماره موبایلتون رو وارد کنید.
    </Text>
    <Text
      mode="regularBody14"
      style={
        Array [
          Array [
            Object {
              "fontFamily": "Dana-Regular",
              "fontSize": 14,
            },
            Object {
              "bottom": 20,
              "color": "#676773",
            },
          ],
        ]
      }
    >
      کد تایید براتون پیامک خواهد شد
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 133.5,
        "justifyContent": "center",
        "width": 750,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "height": 90,
            "width": 675,
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#F9FAFC",
              "overflow": "hidden",
              "paddingHorizontal": 4,
              "position": "absolute",
              "right": 16,
              "top": 18,
              "zIndex": 1,
            },
            Object {
              "transform": Array [
                Object {
                  "translateX": 0,
                },
                Object {
                  "translateY": 0,
                },
                Object {
                  "scale": 1,
                },
              ],
            },
          ]
        }
      >
        <Text
          collapsable={false}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "color": "#676773",
                "fontFamily": "Dana-Regular",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": -0.32,
                "lineHeight": 24,
                "textAlign": "right",
              },
              undefined,
              undefined,
            ]
          }
        >
          شماره همراه
        </Text>
      </View>
      <View
        collapsable={false}
        onGestureHandlerEvent={[Function]}
        onGestureHandlerStateChange={[Function]}
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "backgroundColor": undefined,
              "borderRadius": 8,
              "borderWidth": 1,
              "flexDirection": "row",
              "height": 56,
              "paddingRight": 16,
              "width": 675,
            },
            Object {
              "borderColor": "#676773",
            },
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          errors={Array []}
          keyboardType="phone-pad"
          mode="outlined"
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          rejectResponderTermination={true}
          selectionColor="#D2F2EC"
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "color": "rgba(0,0,0,0.87)",
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": -0.32,
                "lineHeight": 24,
              },
              Object {
                "flex": 9,
                "paddingBottom": undefined,
                "textAlign": "right",
              },
              undefined,
              undefined,
              Object {
                "color": "#676773",
                "fontFamily": "Dana-Regular",
              },
              undefined,
            ]
          }
          textAlignVertical="center"
          title="شماره همراه"
          underlineColorAndroid="transparent"
          value=""
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "row-reverse",
            "height": 34,
            "overflow": "hidden",
            "paddingTop": 8,
            "width": "100%",
          }
        }
      >
        <Text
          mode="regularBody14"
          style={
            Array [
              Array [
                Object {
                  "fontFamily": "Dana-Regular",
                  "fontSize": 14,
                },
                Object {
                  "color": "#FF5A5A",
                  "height": "100%",
                  "marginLeft": 8,
                  "textAlign": "right",
                  "width": "100%",
                },
              ],
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "height": 667,
          "justifyContent": "flex-end",
          "paddingBottom": 66.5,
          "width": 750,
        },
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#DCDDE0",
            "borderColor": "#DCDDE0",
            "borderRadius": 8,
            "borderWidth": 0,
            "flexDirection": "row-reverse",
            "height": 53,
            "justifyContent": "center",
            "maxWidth": 645,
            "minWidth": 675,
            "overflow": "hidden",
            "paddingHorizontal": 4,
          },
        ]
      }
    >
      <Text
        mode="boldBody18"
        numberOfLines={1}
        style={
          Array [
            Array [
              Object {
                "fontFamily": "Dana-Bold",
                "fontSize": 18,
              },
              Object {
                "color": "white",
                "marginHorizontal": 4,
              },
            ],
          ]
        }
      >
        ارسال کد تایید
      </Text>
    </View>
    <Text
      mode="regularSub12"
      style={
        Array [
          Array [
            Object {
              "fontFamily": "Dana-Regular",
              "fontSize": 12,
            },
            Object {
              "color": "#676773",
              "paddingTop": 10,
            },
          ],
        ]
      }
    >
      با ثبت نام و ورود با
      <Text
        mode="boldSub12"
        onPress={[Function]}
        style={
          Array [
            Array [
              Object {
                "fontFamily": "Dana-Bold",
                "fontSize": 12,
              },
              Object {
                "color": "#7766C6",
              },
            ],
          ]
        }
      >
         
        قوانین دنج
         
      </Text>
      موافقت می کنید.
    </Text>
  </View>
</RNCSafeAreaView>
`;
